docs/java-rest/high-level/dataframe/put_data_frame.asciidoc

```diff
@@ -20,6 +20,10 @@ A +{request}+ requires the following argument:
 include-tagged::{doc-tests-file}[{api}-request]
 --------------------------------------------------
 <1> The configuration of the {dataframe-transform} to create
+<2> Whether or not to wait to run deferrable validations until `_start` is called.
```

Comment:
[nit] "Whether or not to wait to run deferrable validations" sounds a bit complicated, but I guess you wanted to avoid "Whether or not to defer deferrable validations until ...". Anyway, I would try to somehow simplify it.

Reply:
Maybe Postpone deferrable validations until _start is called.